src/net.cpp

```diff
@@ -60,6 +63,7 @@
 
 namespace {
     const int MAX_OUTBOUND_CONNECTIONS = 8;
+    const int MAX_FEELER_CONNECTIONS = 1;
```

Comment:
This constant seems not to be used other than to effectively increase MAX_OUTBOUND_CONNECTIONS.